x-pack/plugins/ml/public/application/components/rule_editor/scope_section.js

```diff
@@ -17,8 +17,19 @@ import { ScopeExpression } from './scope_expression';
 import { checkPermission } from '../../capabilities/check_capabilities';
 import { getScopeFieldDefaults } from './utils';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { ML_PAGES } from '../../../../common/constants/ml_url_generator';
+import { useMlUrlGenerator, useNavigateToPath } from '../../contexts/kibana';
 
 function NoFilterListsCallOut() {
+  const mlUrlGenerator = useMlUrlGenerator();
+  const navigateToPath = useNavigateToPath();
+  const redirectToFilterManagementPage = async () => {
+    const path = await mlUrlGenerator.createUrl({
+      page: ML_PAGES.FILTER_LISTS_MANAGE,
```

Comment:
nit: I suppose it'd be more convenient to have `ML_PAGES` as a property of `mlUrlGenerator` to avoid extra static import

Reply:
I think I didn't do this here because of the limitations with  `UrlGeneratorContract<"ML_APP_URL_GENERATOR">`. Let me reach out to Anton and see if we can do something to our contract so others can access it additional properties.